cli/cmd/plugin/ui/web/tanzu-ui/FORMATTING.md

```diff
@@ -16,3 +16,15 @@ Here are some general guidelines for the filenames:
 - Stores - should include `store` between name and suffix (example: App.store.tsx)
 - Actions - should include `actions` between name and suffix (example: App.actions.tsx)
 - Reducers - should include `reducers` between name and suffix (example: App.reducer.tsx)
+
+### Code Formatting and Linting
+
+Code formatting can be applied automatically by leveraging Prettier, which can be accessed via npm scripts.
+
+To run formatting: `/community-edition/cli/cmd/plugin/ui/web/tanzu-ui npm run format`
+
+To run formatting and unit tests (Pre-commit checks): `/community-edition/cli/cmd/plugin/ui/web/tanzu-ui npm run precommit`
```

Comment:
It's pre-commit in the package.json. We could rename either one.

Reply:
You're right thanks for catching...I will update